semgrep/semgrep/nosemgrep.py

```diff
@@ -15,6 +15,7 @@
 
 from semgrep.constants import COMMA_SEPARATED_LIST_RE
 from semgrep.constants import NOSEM_INLINE_RE
+from semgrep.constants import NOSEM_PREVIOUS_LINE_RE
```

Comment:
@brendongo those long list of import are really tedious to write.
Why not import semgrep.constants as constants, and then we can simply use constants.NOSEM_PREVIOUS_LINE_RE so we don't need to list all those constants?

Reply:
pycharm inserts the imports for me and lets me know if they become unused, so it's really not a problem for me.